src/rule/source-location.js

```diff
@@ -72,52 +75,78 @@ report(node, new RuleError("message", {
 `);
         }
 
-        // use {index}
+        const adjustedLoc = this._adjustLoc(node, padding, _backwardCompatibleIndexValue);
+        const adjustedFix = this._adjustFix(node, padding);
+        /*
+        {
+            line,
+            column
+            fix?
+        }
+         */
+        return ObjectAssign({}, adjustedLoc, adjustedFix);
+    }
+
+    _adjustLoc(node, padding, _paddingIndex) {
```

Comment:
fixed in https://github.com/textlint/textlint/commit/6550bf5bdbeba4d13d1e5b802011badff43cc9e2